HoaryFox/Member/CreateTag.cs

```diff
@@ -14,134 +15,167 @@ namespace HoaryFox.Member
     public class CreateTag
     {
         private readonly StbNodes _nodes;
+        private readonly StbSecColumnRc _colRc;
+        private readonly StbSecColumnS _colS;
+        private readonly StbSecBeamRc _beamRc;
+        private readonly StbSecBeamS _beamS;
+        private readonly StbSecBraceS _braceS;
+        private readonly StbSecSteel _secSteel;
         public List<Point3d> Position { get; } = new List<Point3d>();
 
-        public CreateTag(StbNodes nodes)
+        public CreateTag(StbNodes nodes, StbSecColumnRc colRc, StbSecColumnS colS, StbSecBeamRc beamRc, StbSecBeamS beamS, StbSecBraceS braceS, StbSecSteel secSteel)
         {
-            this._nodes = nodes;
+            _nodes = nodes;
+            _secSteel = secSteel;
+            _braceS = braceS;
+            _beamS = beamS;
+            _beamRc = beamRc;
+            _colS = colS;
+            _colRc = colRc;
         }
 
-        public GH_Structure<GH_String> Frame(StbFrame frame, StbSecColumnRc columnRc, StbSecColumnS colS, StbSecBeamRc beamRc, StbSecBeamS beamS, StbSecBraceS braceS, StbSecSteel secSteel)
+        public GH_Structure<GH_String> Frame(StbFrame frame)
```

Comment:
Codacy found an issue: [Rename the parameter 'frame' so that it does not duplicate the method name.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6419994)

Reply:
fix